cron-mm-upgrade-script.sh

```diff
@@ -20,22 +20,23 @@ echo -e "Latest Mattermost version: $latestVersion"
 date=$(date +'%F-%H-%M')
 
 # Establish database variables from Mattermost config.json
-DATABASE=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DriverName')
-DB_USER=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DataSource' | cut -d: -f2 | tr -d '/')
+DB_TYPE=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DriverName')
+DB_USER=$(cat /opt/mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '\"[^ \:]*' | tail -c +2 | head -c -3)
 DB_PASS=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DataSource' | cut -d: -f3 | cut -d@ -f1)
+DB_NAME=$(cat /opt/mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '/.*?' | tail -c +2 | head -c -2)
```

Comment:
This also doesn't work. When I run this, I get almost the whole line of the config for '.SqlSettings.DataSource'. Instead of just the DB_NAME.

The command that worked for me is:
$ cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DataSource' | cut -d? -f1 | cut -d/ -f4
mattermost